### DISCUSSION TITLE
This does not support Azure Private Dns Zone, I suppose ?

### DISCUSSION THREAD

**Discussion by vbakshi123** [shibayan/containerapps-acmebot#57] [Q&A] [answered]
I am assuming [containerapps-acmebot] does not support Azure Private Dns Zone ? The reason being when I try to add the certificate from the dashboard page, the private dns zone does not populate under the drop down list for dnz zones. So, I created a test dns zone in azure DNS instead of Azure private DNS and this time I get the dns zone under the drop down list for dnz zones, but when I try to add the certificate after selecting the container app environment and container app, it throws a pop-up that the AddOrchestratorCertificate process failed- stating that it was expecting the azure nameservers as ns1-38.azure.dns.net and other azure dns servers, but it found null, even though the azure dns zone (not private dns) has got the same NS records. I am assuming, the function app couldn't find a route to the test zone created in Azure DNS, instead of Azure private dns.

**Comment by shibayan** [comment, marked as answer]
There are no plans to add support for anything other than Azure DNS in this project. Ref #20
